mockgcp/mockredis/cluster.go

```diff
@@ -53,7 +53,9 @@ func (r *clusterServer) GetCluster(ctx context.Context, req *pb.GetClusterReques
 		return nil, err
 	}
 
-	return obj, nil
+	retObj := proto.Clone(obj).(*pb.Cluster)
+	retObj.PscConfigs = nil
```

Comment:
Mind sharing more context about this change?
BTW, `PscConfigs` is immutable by Redis service (which makes the single-field update on a list object becomes easier) .

Reply:
I'll add a comment.  pscConfigs is just not included in these response.

Reply:
Added a comment!